src/types/Workload/WorkloadListComponent.ts

```diff
@@ -0,0 +1,147 @@
+import { dicWorkloadType, Workload } from './WorkloadComponent';
+import Namespace from '../Namespace';
+import { ActiveFilter } from '../NamespaceFilter';
+import { removeDuplicatesArray } from '../../utils/Common';
+import { WorkloadListFilters } from './WorkloadListFilters';
+
+export namespace WorkloadList {
+  /** WorkloadItem object to print in the list */
+
+  export class WorkloadItem {
+    namespace: string;
+    workload: Workload.Deployment;
+
+    constructor(ns: string, workload: Workload.Deployment) {
+      this.namespace = ns;
+      this.workload = workload;
+    }
+  }
+  /** WorkloadNamespaceResponse object */
+  export class WorkloadNamespaceResponse {
+    namespace: Namespace;
+    workloads: (Workload.Deployment)[];
+
+    constructor(response: any) {
```

Comment:
I think "response" shouldn't be "any" here, as it comes from the rest API which should provide the right typing.